### DISCUSSION TITLE
1.9.9.3版本浏览画廊及下载页面都只显示左上角一小块

### DISCUSSION THREAD

**Discussion by dogezxy** [xiaojieonly/Ehviewer_CN_SXJ#1577] [Q&A]
rt，好诡异（
![6D0A5B3BC88DD9284A8B06D7F353B236](https://github.com/user-attachments/assets/fefc1e70-c39b-4b40-a5c6-cbeaed7c5c4d)
![8C63B4F995F53A9A30583F9116D6A92B](https://github.com/user-attachments/assets/ecd3764b-2716-47cc-ad32-7f1f8a2f81e4)

**Comment by jsy4141** [comment]
是的我也是，更新之后就成这样了
